sources/Inputs/src/Node.cpp

```diff
@@ -22,15 +22,24 @@ namespace inputs {
 
 std::shared_ptr<Node>
 Node::build(const NodeId& id, const std::shared_ptr<VoltageLevel>& vl, double nominalVoltage, const std::vector<Shunt>& shunts) {
-  auto ret = std::shared_ptr<Node>(new Node(id, vl, nominalVoltage, shunts));
+  bool fictitious = false;
+  auto ret = std::shared_ptr<Node>(new Node(id, vl, nominalVoltage, fictitious, shunts));
   vl->nodes.push_back(ret);
   return ret;
 }
 
-Node::Node(const NodeId& idNode, const std::shared_ptr<VoltageLevel> vl, double nominalVoltageNode, const std::vector<Shunt>& shunts) :
+std::shared_ptr<Node>
+Node::build(const NodeId& id, const std::shared_ptr<VoltageLevel>& vl, double nominalVoltage, bool fictitious, const std::vector<Shunt>& shunts) {
```

Comment:
It is better in that case to put fictitious as the last argument of the list and have the default value "false" than creating another builder

Reply:
Ok. I will also change the private constructor to keep the same order in the arguments.

Reply:
Fixed